### DISCUSSION TITLE
ASE - Group 6

### DISCUSSION THREAD

**Discussion by thepercyj** [thepercyj/Puzzle-Realms#9] [General]
Hello Team, 

Let's use this panel for our latest discussions, post or any project related work starting from today.

**Comment by thepercyj** [comment]
@JUNKAIOU can you send me the solution txt file here or upload it to your branch ?. Thanks

**Reply by Rez27** [reply]
I think he has already added in his branch. Lets work on a script to convert these solutions into an interactive version.
